### PR TITLE
added go ml benchmarks

### PR DESCRIPTION
**Pull request by nikolaydubina**
Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standards) before sending pull request. Thanks!

**Please provide package links to:**

- github.com repo: github.com/nikolaydubina/go-ml-benchmarks
- pkg.go.dev: not applicable
- goreportcard.com: not applicable
- coverage: not applicable

**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order.
- [x] I have an appropriate description with correct grammar.
- [x] I know that this package was not listed before.
- [x] I have added pkg.go.dev link to the repo and to my pull request.
- [x] I have added coverage service link to the repo and to my pull request.
- [x] I have added goreportcard link to the repo and to my pull request.
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:

### CODE REVIEW

**Comment by avelino** [comment]
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body links content check:
- godoc.org or pkg.go.dev: True
- goreportcard.com: False
- coverage: False

> your project is about to be approved, it's under revision, _it may take a few days_

**Comment by nikolaydubina** [comment]
Hi @avelino since this is benchmarks there is no code coverage, goreportcard and pkg.go.dev. 

However, all code is reproduceable and can be run per Makefile instructions.

**Comment by avelino** [comment]
> Hi @avelino since this is benchmarks there is no code coverage, goreportcard and pkg.go.dev.
> 
> However, all code is reproduceable and can be run per Makefile instructions.

review TestAlpha: 
```
    repo_test.go:109: expected 'go-json-benchmark' but actual is 'go-ml-benchmarks'
    repo_test.go:109: expected 'go-ml-benchmarks' but actual is 'go-json-benchmark'
    repo_test.go:113: expected order is:
```

File: README.md [hunk 1]
```diff
@@ -2513,6 +2513,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [go-benchmark-app](https://github.com/mrLSD/go-benchmark-app) - Powerful HTTP-benchmark tool mixed with Аb, Wrk, Siege tools. Gathering statistics and various parameters for benchmarks and comparison results.
 * [go-benchmarks](https://github.com/tylertreat/go-benchmarks) - Few miscellaneous Go microbenchmarks. Compare some language features to alternative approaches.
 * [go-http-routing-benchmark](https://github.com/julienschmidt/go-http-routing-benchmark) - Go HTTP request router benchmark and comparison.
+* [go-ml-benchmarks](https://github.com/nikolaydubina/go-ml-benchmarks) — benchmarks for machine learning inference in Go
 * [go-json-benchmark](https://github.com/zerosnake0/go-json-benchmark) - Go JSON benchmark.
```
Comment (nikolaydubina) [inline README.md:2517]:
```suggestion
* [go-json-benchmark](https://github.com/zerosnake0/go-json-benchmark) - Go JSON benchmark.
* [go-ml-benchmarks](https://github.com/nikolaydubina/go-ml-benchmarks) — benchmarks for machine learning inference in Go
```

File: README.md [hunk 2]
```diff
@@ -2514,6 +2514,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [go-benchmarks](https://github.com/tylertreat/go-benchmarks) - Few miscellaneous Go microbenchmarks. Compare some language features to alternative approaches.
 * [go-http-routing-benchmark](https://github.com/julienschmidt/go-http-routing-benchmark) - Go HTTP request router benchmark and comparison.
 * [go-json-benchmark](https://github.com/zerosnake0/go-json-benchmark) - Go JSON benchmark.
+* [go-ml-benchmarks](https://github.com/nikolaydubina/go-ml-benchmarks) — benchmarks for machine learning inference in Go
```
Comment (nikolaydubina) [inline README.md:2517]:
```suggestion
* [go-ml-benchmarks](https://github.com/nikolaydubina/go-ml-benchmarks) - benchmarks for machine learning inference in Go
```

File: README.md [hunk 3]
```diff
@@ -2514,6 +2514,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [go-benchmarks](https://github.com/tylertreat/go-benchmarks) - Few miscellaneous Go microbenchmarks. Compare some language features to alternative approaches.
 * [go-http-routing-benchmark](https://github.com/julienschmidt/go-http-routing-benchmark) - Go HTTP request router benchmark and comparison.
 * [go-json-benchmark](https://github.com/zerosnake0/go-json-benchmark) - Go JSON benchmark.
+* [go-ml-benchmarks](https://github.com/nikolaydubina/go-ml-benchmarks) - benchmarks for machine learning inference in Go
```
Comment (nikolaydubina) [inline README.md:2517]:
```suggestion
* [go-ml-benchmarks](https://github.com/nikolaydubina/go-ml-benchmarks) - benchmarks for machine learning inference in Go.
```

**Comment by nikolaydubina** [comment]
@avelino lint issues fixed. please check

**Comment by nikolaydubina** [comment]
ping @avelino

**Pull request merged**